src/java.net.http/share/classes/java/net/http/HttpResponse.java

```diff
@@ -748,6 +749,33 @@ public static <T> BodyHandler<T> buffering(BodyHandler<T> downstreamHandler,
                      .buffering(downstreamHandler.apply(responseInfo),
                                 bufferSize);
          }
+
+        /**
+         * {@return a {@code BodyHandler} limiting the number of body bytes
+         * consumed and passed to the given downstream {@code BodyHandler}}
```

Comment:
Would rewording this to:

```
{@return a {@code BodyHandler} that limits the number of body bytes
that are delivered to the given {@code downstreamHandler}}
```
be more concise?

Reply:
Changed in e10771317ef57f80d25be58df77eb5c5b0083b0c.